<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaseManageControl</class>
 <widget class="QWidget" name="CaseManageControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>975</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>542</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>542</width>
    <height>975</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
color:white;
}
QPushButton{
color:black;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_9" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>91</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>91</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>14</number>
         </property>
         <property name="leftMargin">
          <number>22</number>
         </property>
         <property name="topMargin">
          <number>24</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>24</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit_search">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>43</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>43</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>请输入患者姓名/ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_search">
           <property name="minimumSize">
            <size>
             <width>79</width>
             <height>43</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>79</width>
             <height>43</height>
            </size>
           </property>
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>20</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="widget_6" native="true">
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>43</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>43</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>操作管理</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QGridLayout" name="gridLayout">
                    <property name="topMargin">
                     <number>25</number>
                    </property>
                    <property name="horizontalSpacing">
                     <number>22</number>
                    </property>
                    <property name="verticalSpacing">
                     <number>24</number>
                    </property>
                    <item row="2" column="1">
                     <widget class="QPushButton" name="cb_import">
                      <property name="minimumSize">
                       <size>
                        <width>194</width>
                        <height>53</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>194</width>
                        <height>53</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>导入</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>.</normaloff>
                        <disabledoff>.</disabledoff>.</iconset>
                      </property>
                      <property name="iconSize">
                       <size>
                        <width>22</width>
                        <height>22</height>
                       </size>
                      </property>
                      <property name="class" stdset="0">
                       <string>LinearColorButton</string>
                      </property>
                      <property name="TextSize" stdset="0">
                       <string>20</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QPushButton" name="cb_modi">
                      <property name="minimumSize">
                       <size>
                        <width>194</width>
                        <height>53</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>194</width>
                        <height>53</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>修改</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>.</normaloff>
                        <disabledoff>.</disabledoff>.</iconset>
                      </property>
                      <property name="iconSize">
                       <size>
                        <width>22</width>
                        <height>22</height>
                       </size>
                      </property>
                      <property name="class" stdset="0">
                       <string>LinearColorButton</string>
                      </property>
                      <property name="TextSize" stdset="0">
                       <string>20</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <widget class="QPushButton" name="cb_new">
                      <property name="minimumSize">
                       <size>
                        <width>194</width>
                        <height>53</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>194</width>
                        <height>53</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true"/>
                      </property>
                      <property name="text">
                       <string>新建</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>.</normaloff>
                        <disabledoff>.</disabledoff>.</iconset>
                      </property>
                      <property name="iconSize">
                       <size>
                        <width>22</width>
                        <height>22</height>
                       </size>
                      </property>
                      <property name="class" stdset="0">
                       <string>LinearColorButton</string>
                      </property>
                      <property name="TextSize" stdset="0">
                       <string>20</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QPushButton" name="cb_export">
                      <property name="minimumSize">
                       <size>
                        <width>194</width>
                        <height>53</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>194</width>
                        <height>53</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>导出</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>.</normaloff>
                        <disabledoff>.</disabledoff>.</iconset>
                      </property>
                      <property name="iconSize">
                       <size>
                        <width>22</width>
                        <height>22</height>
                       </size>
                      </property>
                      <property name="class" stdset="0">
                       <string>LinearColorButton</string>
                      </property>
                      <property name="TextSize" stdset="0">
                       <string>20</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QPushButton" name="pushButton_image">
                      <property name="minimumSize">
                       <size>
                        <width>194</width>
                        <height>53</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>194</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>图像</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>.</normaloff>
                        <disabledoff>.</disabledoff>.</iconset>
                      </property>
                      <property name="iconSize">
                       <size>
                        <width>22</width>
                        <height>22</height>
                       </size>
                      </property>
                      <property name="class" stdset="0">
                       <string>LinearColorButton</string>
                      </property>
                      <property name="TextSize" stdset="0">
                       <string>20</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QPushButton" name="cb_del">
                      <property name="minimumSize">
                       <size>
                        <width>194</width>
                        <height>53</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>194</width>
                        <height>53</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>删除</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>.</normaloff>
                        <disabledoff>.</disabledoff>.</iconset>
                      </property>
                      <property name="iconSize">
                       <size>
                        <width>22</width>
                        <height>22</height>
                       </size>
                      </property>
                      <property name="class" stdset="0">
                       <string>LinearColorButton</string>
                      </property>
                      <property name="TextSize" stdset="0">
                       <string>20</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0" colspan="2">
                     <widget class="QPushButton" name="cb_open">
                      <property name="minimumSize">
                       <size>
                        <width>194</width>
                        <height>53</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>53</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>进入术前设计</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>.</normaloff>
                        <disabledoff>.</disabledoff>.</iconset>
                      </property>
                      <property name="iconSize">
                       <size>
                        <width>22</width>
                        <height>22</height>
                       </size>
                      </property>
                      <property name="class" stdset="0">
                       <string>LinearColorButton</string>
                      </property>
                      <property name="TextSize" stdset="0">
                       <string>20</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_7" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>160</number>
                  </property>
                  <item>
                   <widget class="QWidget" name="widget" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>43</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>43</height>
                     </size>
                    </property>
                    <property name="autoFillBackground">
                     <bool>false</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <property name="leftMargin">
                      <number>18</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="autoFillBackground">
                        <bool>false</bool>
                       </property>
                       <property name="text">
                        <string>CT预览</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget_5" native="true">
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <property name="spacing">
                      <number>18</number>
                     </property>
                     <property name="leftMargin">
                      <number>18</number>
                     </property>
                     <property name="topMargin">
                      <number>28</number>
                     </property>
                     <property name="rightMargin">
                      <number>15</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QVTKOpenGLNativeWidget" name="widget_dicomWidget" native="true">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>269</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>269</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QSlider" name="verticalSlider_slice">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>28</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>28</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>qvtkopenglnativewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qr.qrc"/>
 </resources>
 <connections/>
</ui>
